docs/aws-eks-tutorial.md

```diff
@@ -0,0 +1,323 @@
+---
+title: Deploy TiDB, a distributed MySQL compatible database, on Kubernetes via AWS EKS
+summary: Tutorial for deploying TiDB on Kubernetes via AWS EKS.
+category: operations
+---
+
+# Deploy TiDB, a distributed MySQL compatible database, on Kubernetes via AWS EKS
+
+## Introduction
+
+This tutorial is designed to be run locally with tools like [AWS Command Line Interface](https://aws.amazon.com/cli/) and [Terraform](https://www.terraform.io/). The Terraform code will ultilize a relatively new AWS service called [Amazon Elastic Container Service for Kubernetes (Amazon EKS)](https://aws.amazon.com/eks).
```

Comment:
I am not exactly sure what do you mean here. I didn't tailor it for anything. I am simply trying to use AWS `EKS` service, and it seems the easiest solution is just to use the default AMI they suggest to use? 
https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html

Reply:
I meant: many people deploy on AWS with a Bastian host running Amazon Linux.  You could simplify the instructions if instead of running locally (with instructions for both Mac OS X + Linux) you provide instructions to run on the Bastian host.  In this way, it would be similar to the GKE tutorial with Cloud Shell always being Debian.

(This is independent of the OS the EKS deploy uses.)